05-query-d1/data/csvtosql.py

```diff
@@ -0,0 +1,41 @@
+# script to convert CSV to SQL to upload to Cloudflare D1
+# after running this python script, run on the command line -> npx wrangler d1 execute {your db name} --remote --file=data/data.sql
+import csv
+from pathlib import Path
+
+p = Path(__file__).with_name('quotes.csv')
+
+# Path to the uploaded CSV file
+csv_file_path = 'quotes.csv'
+
+# Initialize list to store SQL insert statements
+sql_statements = []
+
+# Open and read the CSV file
+with p.open('r') as file:
+    reader = csv.reader(file)
+    header = next(reader)  # Skip header row if present
+    
+    # Loop through each row in the CSV
+    for i, row in enumerate(reader, start=1):
+        # Extract quote and author, assuming they are in the first two columns
+        quote, author = row[0].replace("'", "''"), row[1].replace("'", "''")
+        
+        # Create the SQL insert statement
+        sql_statements.append(f"INSERT INTO qtable (id, quote, author) VALUES ('{i}', '{quote}', '{author}');")
```

Comment:
This will probably catch most SQL injection attempts, but not all. Maybe it doesn't matter for this script, but it sets a bad precedent if someone copies it into their code without being careful.

Instead of including this script which converts CSV to SQL, then asking the user to run it, maybe we should host the SQL itself somewhere and ask the user to download it?